pkg/osquery/Dockerfile

```diff
@@ -48,6 +48,7 @@ USER root
 #collect all files and prepare them into a tar file
 RUN mkdir -p /opt/osquery/lenses /opt/osquery/yara /opt/osquery/extensions \
  && cp /home/"$OSQUERY_BUILD_USER"/osquery/build/osquery/osqueryd /opt/osquery \
+ && cp /home/"$OSQUERY_BUILD_USER"/osquery/build/osquery/osqueryi /opt/osquery/ \
```

Comment:
Also **/home/"$OSQUERY_BUILD_USER"/osquery/build/osquery/osqueryi** doesn't actually exists. The osqueryd binary is the same as osqueryi binary. So osqueryd needs to be copied at both places. The correct cp statement should be **cp /home/"$OSQUERY_BUILD_USER"/osquery/build/osquery/osqueryd /opt/osquery/osqueryi**

Reply:
or maybe a hardlink? if it's really the same file, symlinking or hardlinking would seem the right thing to do to me.